packages/fast-tooling-react/src/form/form.props.ts

```diff
@@ -25,42 +25,16 @@ export type FormTag = "form" | "div";
  */
 export interface FormProps {
     /**
-     * The optional class name
+     * The message system
+     * used for sending data to the message system
      */
-    className?: string;
```

Comment:
You might consider abstracting this to a interface that your form knows how to interact with. That way this doesn't necessarily need to be a web-worker and is generally more extensible.

Reply:
This is a really good point that I hadn't thought about, I've filed an issue to be tackled next: https://github.com/microsoft/fast-dna/issues/2668